### ISSUE TITLE
v0.52.1 is not compatible with latest Nib

### ISSUE THREAD

**Issue by miu-zsolt** [stylus/stylus#1995]
Hello! We have an until now well working build system based on stylus. Our relevant dependencies:
    "gulp-stylus": "^2.0.1",
    "nib": "^1.1.0",
And gulp-stylus depends on this stylus version:
    "stylus": "^0.52.0",
It was perfectly working with 0.52.0 however after releasing 0.52.1 our new build all failed with the following error:

```
Potentially unhandled rejection [2] ParseError: /opt/app/node_modules/nib/lib/nib/normalize/base.styl:5:5
   1| // Based in Nicolas Gallagher's git.io/normalize
   2| normalize-base()
   3|   html // Prevent iOS text size adjust after orientation change.
   4|     font-family sans-serif
   5|     -ms-text-size-adjust 100%
----------^
   6|     -webkit-text-size-adjust 100%
   7|   body
   8|     margin 0

unexpected "indent"

    at Parser.error (/opt/app/node_modules/gulp-stylus/node_modules/stylus/lib/parser.js:257:11)
    at Parser.stmt (/opt/app/node_modules/gulp-stylus/node_modules/stylus/lib/parser.js:789:32)
    at Parser.statement (/opt/app/node_modules/gulp-stylus/node_modules/stylus/lib/parser.js:660:21)
    at Parser.block (/opt/app/node_modules/gulp-stylus/node_modules/stylus/lib/parser.js:835:21)
    at Parser.functionDefinition (/opt/app/node_modules/gulp-stylus/node_modules/stylus/lib/parser.js:1713:21)
    at Parser [as function] (/opt/app/node_modules/gulp-stylus/node_modules/stylus/lib/parser.js:1654:18)
    at Parser.stmt (/opt/app/node_modules/gulp-stylus/node_modules/stylus/lib/parser.js:753:26)
    at Parser.statement (/opt/app/node_modules/gulp-stylus/node_modules/stylus/lib/parser.js:660:21)
    at Parser.parse (/opt/app/node_modules/gulp-stylus/node_modules/stylus/lib/parser.js:234:25)
    at Evaluator.importFile (/opt/app/node_modules/gulp-stylus/node_modules/stylus/lib/visitor/evaluator.js:74:20)
```

Could you please fix this issue?

**Comment by alexeymorozov** [comment]
:+1:

**Comment by Panya** [comment]
Thank for the report. I've fixed this bug in the `dev` branch. We will release a new version soon, sorry for the inconvenience.

**Comment by miu-zsolt** [comment]
Thank you @Panya that was quick! I'm waiting for the new version.

**Comment by Panya** [comment]
Should be fixed in `0.52.2`.
